docs/pipelines/edit-pipeline.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Edit a Pipeline
+intro: Editing a pipeline is crucial as it provides a clear and concise way to identify and differentiate between pipelines in your environment.
+links:
+    overview:
+    quickstart:
+    previous:
+    next:
+    guides:
+    related:
+    featured:
+---
+
+1. On Devopness, navigate to a project then select an environment
+2. Find the `Applications` card
+3. Click `View` in the `Applications` card to see a list of existing `Applications`
+4. In the list of applications find the application with the pipeline you want to edit and click `DETAILS`
+5. On the upper-right corner of the applications details view click `SETTINGS`
+6. Use the drop-down menu to choose `PIPELINES`
+8. In the list of pipelines find the pipeline you want to edit and click `DETAILS`
+9. On the upper-right corner of the key details click `...`
+10. Use the drop-down menu to choose `EDIT`
+11. Follow the instructions in the pipeline edit form then click `REMOVE`
```

Comment:
@plantaz, why a post named "Edit a Pipeline" leads the user to removing a pipeline?

Reply:
Done, thanks!